### DISCUSSION TITLE
Parse error: syntax error, unexpected end of file, please help!

### DISCUSSION THREAD

**Discussion by ghost** [PHPMailer/PHPMailer#2490] [Q&A]
I'm working with PHPmailer 6.5.1 as installed via Composer 1.  This is a truncated bit of code showing where I'm trying to send mail.  There is also an HTML portion below this.   When testing,, I get "Parse error: syntax error, unexpected end of file" and it shows the line number of the last line of the HTML.  Can anyone help?  I'm con fused!  Thanks!

https://pastebin.com/2LVr5ut8

**Comment by Synchro** [comment]
The code that you posted doesn't cause a syntax error, so there's nothing to diagnose. This will be something like a missing `}` off the end of an `if` statement, a missing `;`, or similar. No doubt an IDE would make it very obvious where the problem is.

**Reply by ghost** [reply]
That's what I was thinking.  However, I decided to stick with the built in mail function as it works for my purposes and I only use it to send an occasional e-mail to one specific address.

Thanks for your time!
